docs/workers.md

```diff
@@ -146,6 +150,99 @@ plain HTTP endpoint on port 8083 separately serving various endpoints, e.g.
 Obviously you should configure your reverse-proxy to route the relevant
 endpoints to the worker (`localhost:8083` in the above example).
 
+---
```

Comment:
Sorry to be annoying but I don't think the full configuration options should go in the worker doc, but should stay in the configuration manual and be referenced from the doc.

Reply:
Perhaps a "subchapter" then, something clearly saying "This doesn't belong in the same configuration file as all that other stuff you just read about above?"

Reply:
I think adding a section with a header with the information about which config to put these options and any other relevant info should be fine, i.e. how the other sections are defined:
https://github.com/matrix-org/synapse/blob/755bfeee3a1ac7077045ab9e5a994b6ca89afba3/docs/usage/configuration/config_documentation.md?plain=1#L75